script/generate-jsx.js

```diff
@@ -38,10 +38,26 @@ fs.writeFileSync(
         // @ts-ignore Hush, `2021` is fine.
         {sourceType: 'module', ecmaVersion: 2021}
       ),
-      {runtime: 'automatic', importSource: '.'}
+      {runtime: 'automatic', importSource: 'xastscript'}
     )
     // @ts-expect-error Some bug in `to-js`
-  ).value.replace(/\/jsx-runtime(?=["'])/g, './lib/runtime.js')
+  ).value
+)
+
+fs.writeFileSync(
+  path.join('test', 'jsx-build-jsx-automatic-development.js'),
+  toJs(
+    // @ts-expect-error it’s a program.
+    buildJsx(
+      // @ts-expect-error Acorn nodes are assignable to ESTree nodes.
+      Parser.extend(acornJsx()).parse(
+        doc.replace(/'name'/, "'jsx (estree-util-build-jsx, automatic)'"),
+        {sourceType: 'module', ecmaVersion: 2021}
```

Comment:
This might error (it just did for me on the above ones, see the `ts-ignore` for more info

Reply:
Yeah I had to resolve some merge conflicts here. Not sure what happened upstream.